Dockerfile

```diff
@@ -5,6 +5,20 @@ WORKDIR /code
 
 COPY entrypoint.sh /entrypoint.sh
 COPY node-build.sh /node-build.sh
-RUN chmod +x /entrypoint.sh
-RUN chmod +x /node-build.sh
-CMD [ "bash", "/entrypoint.sh" ]
+
+# set execute and install browsers
+RUN chmod +x /entrypoint.sh && \
+    chmod +x /node-build.sh && \
+    apt-get update && \
+    apt-get install -y chromium firefox-esr
+
+# install sonar-scanner 
+RUN SONAR=4.5.0.2216-linux && \
```

Comment:
This will always install the exact same version, why not download the zip and add it to the container so we know it's always the legit version. (prevents things like dns poisoning)

Reply:
do remember to add it, install it and delete it before finishing.

Reply:
Very true! saves time, security issues and above all, the download might not always be avaiable
Downside is that it will take 50M on the git repo :-(